packages/design-system/src/components/web-components/ds-tooltip/ds-tooltip.test.tsx

```diff
@@ -0,0 +1,152 @@
+    expect(asFragment()).toMatchSnapshot();
+  });
+
+  it('renders title and contentHeading when passed in as slots', () => {
+    renderTooltip(propsWithSlots);
+    const tooltipTrigger = screen.getByText(customTooltipText);
+    expect(tooltipTrigger).toHaveTextContent(customTooltipText);
+    expect(tooltipTrigger).toHaveTextContent(customHeadingText);
+  });
+
+  it('renders default trigger icon', () => {
+    renderTooltip();
+    const triggerEl = screen.queryByTestId('ds-tooltip-icon');
+    expect(triggerEl).toMatchSnapshot();
+  });
+
+  it('renders custom trigger component', () => {
+    renderTooltip({
+      component: 'a',
+      children: childText,
+    });
+    const triggerEl = screen.queryByText(childText);
+    expect(triggerEl).toMatchSnapshot();
+  });
+
+  it('renders inverse tooltip', () => {
+    const { asFragment } = renderTooltip({ inversed: 'true' });
+    expect(asFragment()).toMatchSnapshot();
+  });
+
+  it('renders dialog tooltip', () => {
+    renderTooltip({ dialog: 'true', children: childText });
+    const tooltipTrigger = screen.getByText(childText);
+    userEvent.click(tooltipTrigger);
+    const contentEl = screen.queryByRole('dialog');
+    expect(contentEl).not.toBeNull();
+    expect(contentEl).toMatchSnapshot();
+  });
+
+  it('closes tooltip when trigger focus is lost', async () => {
+    jest.useFakeTimers();
+    const { container } = renderTooltip();
+    const tooltip = container.querySelector('.ds-c-tooltip');
+
+    userEvent.tab();
+    act(() => {
+      jest.advanceTimersByTime(100);
+    });
+    expect(tooltip).toHaveClass('ds-c-tooltip-enter');
+
+    userEvent.tab();
+    act(() => {
+      jest.advanceTimersByTime(100);
+    });
+    expect(tooltip).toHaveClass('ds-c-tooltip-exit');
+  });
+
+  it('renders a close button', () => {
+    renderTooltip({
+      dialog: 'true',
+      'show-close-button': 'true',
+      title: customTooltipText,
+    });
+    const closeButton = screen.getByLabelText('Close', { selector: 'button' });
+    expect(closeButton).toBeDefined();
+  });
+
+  it('renders heading element', () => {
+    renderTooltip({
+      dialog: 'true',
+      'content-heading': 'Tooltip heading content',
+      title: customTooltipText,
+    });
+    const tooltipTrigger = screen.getByText(customTooltipText);
+    userEvent.click(tooltipTrigger);
+    const contentEl = screen.queryByRole('dialog');
+    expect(contentEl).toMatchSnapshot();
+  });
+
+  it('renders heading element and close button', () => {
+    renderTooltip({
+      dialog: 'true',
+      'content-heading': customHeadingText,
+      'show-close-button': 'true',
+      title: customTooltipText,
+    });
+    const tooltipTrigger = screen.getByText(customTooltipText);
+    userEvent.click(tooltipTrigger);
+    const contentEl = screen.queryByRole('dialog');
+    expect(contentEl).toMatchSnapshot();
+  });
+
+  it('should return focus back to trigger when closed', () => {
```

Comment:
Is it work adding another expectation around having the tooltipContent be closed?

https://github.com/CMSgov/design-system/blob/ba8a9262992e91a3111900dd1909b51e6ee6f374/packages/design-system/src/components/Tooltip/Tooltip.test.tsx#L113